flang/lib/Lower/OpenMP.cpp

```diff
@@ -3189,13 +3208,15 @@ static void genOMP(Fortran::lower::AbstractConverter &converter,
     if ((llvm::omp::allTargetSet & llvm::omp::loopConstructSet)
             .test(ompDirective)) {
       validDirective = true;
-      genTargetOp(converter, eval, semanticsContext, currentLocation,
-                  loopOpClauseList, ompDirective, /*outerCombined=*/true);
+      genTargetOp(converter, eval, semanticsContext, /*genNested=*/false,
```

Comment:
`target` has a structured nested block. Can this nested block have additional OpenMP constructs? And if `genNested=false`, then are we relying on the bridge to lower those nested evaluations?

Reply:
The `targetOp` here is a part of a composite OpenMP directive, so the only nested constructs possible are those nested in the provided `loopConstruct`.